RepliCare:
        joint_state_controller:
                type: joint_state_controller/JointStateController
                publish_rate: 50

        left_shoulder_yaw_position_controller:
                type: position_controllers/JointPositionController
                joint: left_shoulder_yaw
                pid: {p: 100.0, i: 0.01, d: 10.0}
        left_shoulder_pitch_position_controller:
                type: position_controllers/JointPositionController
                joint: left_shoulder_pitch
                pid: {p: 100.0, i: 0.01, d: 10.0}
        left_shoulder_roll_position_controller:
               type: position_controllers/JointPositionController
               joint: left_shoulder_roll
               pid: {p: 100.0, i: 0.01, d: 10.0}
        left_elbow_position_controller:
               type: position_controllers/JointPositionController
               joint: left_elbow
               pid: {p: 100.0, i: 0.01, d: 10.0}
        right_shoulder_yaw_position_controller:
               type: position_controllers/JointPositionController
               joint: right_shoulder_yaw
               pid: {p: 100.0, i: 0.01, d: 10.0}
        right_shoulder_pitch_position_controller:
               type: position_controllers/JointPositionController
               joint: right_shoulder_pitch
               pid: {p: 100.0, i: 0.01, d: 10.0}
        right_shoulder_roll_position_controller:
               type: position_controllers/JointPositionController
               joint: right_shoulder_roll
               pid: {p: 100.0, i: 0.01, d: 10.0}
        right_elbow_position_controller:
               type: position_controllers/JointPositionController
               joint: right_elbow
               pid: {p: 100.0, i: 0.01, d: 10.0}
